helm/korifi/values.schema.json

```diff
@@ -468,6 +468,11 @@
       "required": ["include", "jobTTL"],
       "type": "object"
     },
+    "korifiGatewayNamespace": {
```

Comment:
After updating the schema json, please make sure to run `make generate` to update the `README.helm.md` doc with the new value